config/config_test.go

```diff
@@ -37,50 +38,33 @@ func TestNewConfigBadFileConfig(t *testing.T) {
 	filename, _ := utils.CreateTempFileWithContent(jsonConfig, "test-*.yaml")
 	defer os.Remove(filename)
 	err := NewConfigFromFile(filename)
-	if err != nil {
-		t.Errorf("Failed!")
-	}
+	assert.NoError(t, err)
 }
 
 func TestNewConfigConfig(t *testing.T) {
 	filename, _ := utils.CreateTempFileWithContent(yamlConfig, "test-*.yaml")
 
 	err := NewConfigFromFile(filename)
-	if err != nil {
-		t.Errorf("Failed!")
-	}
-
-	if len(FTWConfig.TestOverride.Ignore) == 0 {
-		t.Errorf("Failed! Len must be > 0")
-	}
-
-	if reflect.ValueOf(FTWConfig.TestOverride.Input).IsZero() {
-		t.Errorf("Failed! Input must not be empty")
-	}
+	assert.NoError(t, err)
+	assert.Greater(t, len(FTWConfig.TestOverride.Ignore), 0, "Failed! Len must be > 0")
+	assert.NotEmpty(t, reflect.ValueOf(FTWConfig.TestOverride.Input), "Failed! Input must not be empty")
```

Comment:
Is this needed?

Reply:
Probably not. Let me see if things work without it.